lib/fb-user-filestore-client.js

```diff
@@ -0,0 +1,173 @@
+const FBJWTClient = require('@ministryofjustice/fb-jwt-client-node')
+class FBUserFileStoreClientError extends FBJWTClient.prototype.ErrorClass {}
+
+const util = require('util')
+const fs = require('fs')
+const readFile = util.promisify(fs.readFile)
+
+// endpoint urls
+const endpoints = {
+  fetch: '/service/:serviceSlug/user/:userId/:fingerprint',
+  store: '/service/:serviceSlug/user/:userId'
+}
+
+// defaults - can be overridden in constructor
+let defaultMaxSize = 10 * 1024 * 1024 // 10Mb
+let defaultExpires = '28d'
```

Comment:
i've implemented this as `28` i can change my end to support the `d` suffix

Reply:
As per our conversation, it makes much more sense to just handle this as days and as a number. Done in dea055b